src/components/Tag/Tag.stories.tsx

```diff
@@ -20,9 +20,9 @@ Source: https://designsystem.digital.gov/components/tag/
 export const DefaultTag = (): React.ReactElement => <Tag>My Tag</Tag>
 
 export const CustomBg = (): React.ReactElement => (
-  <Tag background="#e44608">My Tag</Tag>
+  <Tag background="#d83933">My Tag</Tag>
```

Comment:
Totally a nit, but this could be a different color (that's also accessible of course). Right now this is the same as `bg-secondary` so there's no visual difference looking at the CustomBg and CustomClass stories.

Reply:
I remember doing that on purpose, but I forget why 😅

Reply:
i think because it _looked_ like bg-secondary, but it wasn't, so I was trying to keep it within the palette